api/service_routes/room-booking.yaml

```diff
@@ -5,16 +5,151 @@ info:
 servers:
   - url: https://room-booking.reserva.emilharvey.ca
 paths:
-  /test:
+  /bookings/{id}:
+    parameters:
+    - name: id
+      in: path
+      required: true
+      schema:
+        type: integer
+        minimum: 1
+      description: The user ID
     get:
+      tags: 
+        - Booking
+      description: Retrieves information about a specific booking identified by the `id` parameter.
       responses:
         '200':
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Booking"
```

Comment:
Let's update this so that the response is:
```
{
    "booking": {
        "id": ...,
        "created-by": ...,
        ...
    }
}
```

Reply:
We should do this for all responses

Reply:
![image](https://user-images.githubusercontent.com/80491585/228697401-a5ea7e62-1e7a-43ee-aea4-195fe0e0fcdb.png)
Updated